File: contests/nye18/p3/p3_solution.py
n=int(input())
max_area=0
px=0
py=0
for i in range(n):
    x,y,w,h=map(int,input().split())
    area=w*h
    if area > max_area:
        max_area=area
        px=x
        py=y
    
print(px,py)

File: contests/nye18/p3/p3_solution.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;
    
    int max_area=0;
    int px,py;
    for (int i = 0; i < n; ++i) {
        int x,y,w,h;
        std::cin >> x >> y >> w >> h;
        int area = w*h;
        if (area > max_area) {
            max_area = area;
            px=x;
            py=y;
        }
    }
    
    std::cout << px << " " << py << "\n";
}